native-authentication/src/test/java/com/okta/nativeauthentication/form/transformer/SignInTitleTransformerTest.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2022-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.nativeauthentication.form.transformer
+
+import com.google.common.truth.Truth.assertThat
+import com.okta.nativeauthentication.RealIdxResponseTransformer
+import com.okta.nativeauthentication.form.Element
+import com.okta.nativeauthentication.form.Form
+import com.okta.nativeauthentication.utils.IdxResponseFactory
+import com.okta.testing.network.NetworkRule
+import kotlinx.coroutines.runBlocking
+import org.junit.Rule
+import org.junit.Test
+
+internal class SignInTitleTransformerTest {
+    @get:Rule val networkRule = NetworkRule()
+
+    private val idxResponseFactory = IdxResponseFactory(networkRule)
+
+    private fun setup(json: String): Form = runBlocking {
```

Comment:
Nit: Should we rename it to something else since setup is usually used for ```@Before``` annotated functions? How about getFormFromJson? Or setupForm